includes/wikia/Wikia.php

```diff
@@ -1106,6 +1106,34 @@ static public function onSkinTemplateOutputPageBeforeExec(SkinTemplate $skinTemp
 		$tpl->set( 'thisquery', $skinTemplate->thisquery );
 
 		$robotPolicy = Wikia::getEnvironmentRobotPolicy( $skinTemplate->getRequest() );
+		$request = $skinTemplate->getRequest();
+		$setNofollow = false;
+		if( $title->inNamespaces( NS_SPECIAL, NS_USER, NS_USER_TALK, NS_HELP) ) {
+			$setNofollow = true;
+		} else {
+			$noindexParams = [
+				'action',
+				'feed',
+				'from',
+				'oldid',
+				'printable',
+				'redirect',
+				'useskin',
+				'uselang',
+				'veaction'
+			];
+			foreach($noindexParams as $paramName ){
+				if( array_key_exists( $paramName, $request->getQueryValues() ) ) {
+					$setNofollow = true;
+					break;
+				}
+			}
+		}
+
+		if( $setNofollow ){
+			$out->setRobotPolicy( "noindex, nofollow" );
```

Comment:
should it be `noindex, nofollow` or just `noindex`?

Reply:
I think that noindex, nofollow . Some of those pages already had them set.